packages/api/src/controllers/taskController.js

```diff
@@ -279,7 +279,7 @@ const taskController = {
           const { assignee_user_id, task_id, taskType } = result;
           await sendTaskNotification(
             [assignee_user_id],
-            null,
+            assignee_user_id,
```

Comment:
Maybe we should have a call about this? From my view, these changes relocate my concern instead of addressing it. Based on the parameter names and doc comment for sendTaskNotification, I read this code as saying that the assignee will sent a notification to themself.
The null-- meaning there is no "user sender" was appropriate-- my concern was sending some user identification to the front end (via a database record) in spite of this.
Can we instead conditionally exclude the user name from the `variables` to be put in the notification body?